packages/sdk/src/Incentives/ERC20PeggedIncentive.ts

```diff
@@ -0,0 +1,490 @@
+import {
```

Comment:
might be good to hook this into the rest of the sdk similar to #286

Reply:
Yeah I should do that before we merge

Reply:
Can you double check I got everything and approve?

Reply:
Yeah this isn't hooked into BoostCore.ts or exported via index.ts or anything like the link below but it looks okay on its own.

https://github.com/boostxyz/boost-protocol/blob/79db6e6160a3427e9f0dadf3be59397b2f254725/packages/sdk/src/BoostCore.ts#L1368-L1373